src/python/tensorflow_cloud/core/tests/unit/containerize_test.py

```diff
@@ -30,14 +30,17 @@
     VERSION = "latest"
 
 
+@mock.patch("tensorflow_cloud.core.containerize.requests")
```

Comment:
you should not mock indirectly-imported module. You should mock `requests` package directly

Reply:
Updated.